package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
   },
   "engines": {
-    "node": "8.9.0",
+    "node": "8.9.1",
```

Comment:
Is the agreement to use version `8.9.0` or this version?

Reply:
its 8.9.0 on my local branch here.... i have no idea why it changes to 8.9.1
But we all agreed to use 8.9.0

Reply:
I just resolved it now....